CHANGELOG.md

```
@@ -1,12 +1,18 @@
+# [1.2.0](https://github.com/fvena/handler-error/compare/v1.1.0...v1.2.0) (2025-01-17)
+
+### Features
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)